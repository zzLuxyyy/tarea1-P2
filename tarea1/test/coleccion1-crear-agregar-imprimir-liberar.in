crearColeccion

# creo libro para ingresarlo a la colección
crearFecha 26/06/1997
crearLibro 171819202 "Harry Potter y la piedra filosofal" "J.K." "Rowling" "Un niño mago descubre su destino en la escuela Hogwarts" 6

agregarEnColeccion
imprimirColeccion

liberarColeccion
Fin
