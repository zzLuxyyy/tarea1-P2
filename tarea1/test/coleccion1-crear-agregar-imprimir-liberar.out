1>2>#  creo libro para ingresarlo a la colección.
3>4>5>6>Libro 171819202
Título: Harry Potter y la piedra filosofal
Autor: J.K. Rowling
Descripción: Un niño mago descubre su destino en la escuela Hogwarts
Género: 6
Fecha de edición: 26/6/1997
7>8>Fin.
